src/main/java/org/itsallcode/openfasttrace/importer/legacytag/LegacyTagImporterConfig.java

```diff
@@ -24,24 +24,32 @@
 
 import static java.util.Collections.emptyList;
 
+import java.nio.file.Path;
 import java.util.List;
 
 public class LegacyTagImporterConfig
```

Comment:
Add Java doc to all public classes and methods please.

Reply:
✔️